prov/rxm/src/rxm_cq.c

```diff
@@ -61,30 +61,30 @@ static const char *rxm_cq_strerror(struct fid_cq *cq_fid, int prov_errno,
 	return fi_cq_strerror(rxm_ep->msg_cq, prov_errno, err_data, buf, len);
 }
 
-/* Get a match_iov derived from iov whose size matches given length */
 static int rxm_match_iov(const struct iovec *iov, void **desc,
 			 uint8_t count, uint64_t offset, size_t match_len,
 			 struct rxm_iov *match_iov)
```

Comment:
nit: IMO, the term 'match' in the function and parameter names could be replaced.  The function is formatting or copying an iov, not really matching it.  This looks like it could move into ofi_iov.h as a common function, with optimizations similar for ofi_copy_to_iov()